_summaries/image-gif.md

```diff
@@ -0,0 +1,6 @@
+---
+name: "tiff"
```

Comment:
I guess you mean `gif`?

Reply:
Turns out I forgot to adjust the metadata all around.